src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/servicebus/ServiceBusHelper.java

```diff
@@ -19,22 +16,15 @@
 import static uk.gov.hmcts.reform.bulkscanprocessor.model.out.msg.MsgLabel.TEST;
 
 @Component
-// Service bus clients are expensive to create and thread safe so perfectly reusable.
-// With prototype scope we can have more than 1 client instance. Also injection happens
-// when the supplier is accessed for the first time (not at application startup), this
-// allows to avoid mocking the supplier in every single integration test.
-@Scope(
-    value = ConfigurableBeanFactory.SCOPE_PROTOTYPE, // every other call to return bean will create new instance
-    proxyMode = ScopedProxyMode.TARGET_CLASS // allows prototyping
-)
+@Lazy
 public class ServiceBusHelper {
 
     private final IQueueClient sendClient;
 
     private final ObjectMapper objectMapper;
 
-    ServiceBusHelper(QueueClientSupplier queueClientSupplier, ObjectMapper objectMapper) {
-        this.sendClient = queueClientSupplier.get();
+    ServiceBusHelper(IQueueClient queueClient, ObjectMapper objectMapper) {
```

Comment:
Sooo there is a lifecycle problem here 
The queueClient does not have a Shutdown hook 
So we need to put in a @PreDestroy annotated method in here to shutdown the client ...
Maybe make this AutoClosable and put a @PreDestroy on the close method.
❌

Reply:
Unfortunately your suggestion would not have any effect as close really happen on finalization as it is obvious if you look at 
https://tools.hmcts.net/jira/browse/RPE-797.
This means that your predestroy hook would not have any effect as application shutdown does not necessarily imply gc and finalisation.